templates/verify.html

```diff
@@ -31,6 +31,14 @@
     </style>
   </head>
   <body leftmargin="0" marginwidth="0" topmargin="0" marginheight="0" offset="0" style="background-color:#F2F2F2;width:100% !important;height:100% !important;padding:0;margin:0;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%;">
+    <div itemscope itemtype="http://schema.org/EmailMessage">
+      <div itemprop="potentialAction" itemscope itemtype="http://schema.org/ViewAction">
```

Comment:
I don't know nothing about HTML emails, but do we want these to be HTTP or HTTPS? Both seem to resolve correctly on schema.org. Wasn't sure if there would be potential issues w/ an HTTPS mail client, or probably not since I'm guessing these aren't downloading specific resources.